src/scripts/header.js

```diff
@@ -0,0 +1,86 @@
+const dropdown = document.querySelector(".nav-stories");
+const dropdownValue = document.querySelector(".stories-dropdown");
+const nav = document.querySelector("nav");
+const arrow = document.querySelector(".nav-stories i");
+
+const hamburgerMenu = document.querySelector("nav .menu");
+const navbarLeft = document.querySelector("nav .navbar-left");
+const overlay = document.querySelector(".overlay");
+
+let pageScrollingDown = false;
+let pageScrollingUp = false;
+
+let oldValue = 0;
+let newValue = 0;
+
+document.addEventListener("scroll", () => {
+  // checking scroll direction
+  newValue = window.pageYOffset;
+
+  if (oldValue < newValue) {
+    pageScrollingDown = true;
+    pageScrollingUp = false;
+  } else if (oldValue > newValue) {
+    pageScrollingDown = false;
+    pageScrollingUp = true;
+  }
+
+  oldValue = newValue;
+
+  // updating nav position/styling as per scroll direction & offset
+  if (pageScrollingDown) {
+    if (
+      (document.body.scrollTop > 38 ||
```

Comment:
You should add the `body` and `document` elements in variables to avoid looking them up everytime you want to update them